src/app/core/services/edit/edit.service.ts

```diff
@@ -208,7 +208,7 @@ export class EditService {
           if (items.length === 1) {
             const item = items[0];
             if (item instanceof RecordVO) {
-              this.dataService.downloadFile(item);
+              this.dataService.downloadFile(item, '');
```

Comment:
Why not pass the type here? I'm not totally sure what the edit service is so I'm sure there's a good reason, just trying to follow.

Reply:
Good idea, this way i could refactor the download function as well

Reply:
Although isn't the type nested inside the FileVO? Maybe I could just pass "type.format.original" because i think that case only downloads original files. What do you think?

Reply:
Because that's what i was trying to achieve with the empty string :)

Reply:
I'm fine with however you want to do it! I was just curious what your reasoning was.